### PR TITLE
matterircd: Add version 0.22.0

### PR DESCRIPTION
**Pull request by 42wim**

### CODE REVIEW

File: bucket/matterircd.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "bin": "matterircd.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-64bit.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-32bit.exe"
```
Comment (Ash258) [inline bucket/matterircd.json:24]:
```suggestion
                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-32bit.exe#/matterircd.exe"
```

File: bucket/matterircd.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            "url": "https://github.com/42wim/matterircd/releases/download/v0.22.0/matterircd-0.22.0-windows-32bit.exe#/matterircd.exe",
+            "hash": "c9794bc898c97e236f681a359a41fdc9845c63e8e7068c5c072c929fbba530a4"
+        }
+    },
+    "bin": "matterircd.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-64bit.exe"
```
Comment (Ash258) [inline bucket/matterircd.json:21]:
```suggestion
                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-64bit.exe#/matterircd.exe"
```

File: bucket/matterircd.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.22.0",
+    "description": "Connect to your mattermost or slack using your IRC-client of choice.",
```
Comment (Ash258) [inline bucket/matterircd.json:3]:
```suggestion
    "description": "Connects to mattermost or slack using IRC-client.",
```

**Review by Ash258 — COMMENTED** [review]
- Add hash extraction from $baseurl/checksums

**Review by Ash258 — CHANGES_REQUESTED** [review]
Above

**Comment by 42wim** [comment]
Thanks, fixes applied

File: bucket/matterircd.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-64bit.exe#/matterircd.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/42wim/matterircd/releases/download/v$version/matterircd-$version-windows-32bit.exe#/matterircd.exe"
+            }
+        }
```
Comment (Ash258) [inline bucket/matterircd.json:26]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**